### PR TITLE
update sample node DICT-235

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: gdcdictionary/schemas/sample.yaml [hunk 1]
```diff
@@ -1013,6 +1020,7 @@ properties:
       - Xenograft
       - Unknown
       - Not Reported
+      - New Primary
```
Comment (wwysoc2) [inline gdcdictionary/schemas/sample.yaml:1023]: Alphabetize this one.

File: gdcdictionary/schemas/sample.yaml [hunk 2]
```diff
@@ -843,49 +847,52 @@ properties:
...
+      - Pleural Effusion
+      - Saliva
+      - Serum
+      - Solid Tissue
+      - Sorted Cells
+      - Sputum
+      - Whole Bone Marrow
+      - Unknown
+      - Not Reported
+      - Bone Marrow NOS
```
Comment (wwysoc2) [inline gdcdictionary/schemas/sample.yaml:890]: Alphabetize this and below.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]
I think I only found some things that need alphabetizing.

File: gdcdictionary/schemas/sample.yaml (hunk 2 above)
Comment (bullenc) [reply gdcdictionary/schemas/sample.yaml]: alphabetized 👍

File: gdcdictionary/schemas/sample.yaml (hunk 1 above)
Comment (bullenc) [reply gdcdictionary/schemas/sample.yaml]: also alphabetized

**Review by wteouchicago — APPROVED** [review]
looks good

**Review by ProfOak — APPROVED** [review]

**Review by ProfOak — APPROVED** [review]

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**